Documentation/production-users.md

```diff
@@ -145,3 +145,17 @@ In [hyper.sh][hyper.sh], the container service is backed by [hypernetes][hyperne
 
 [hypernetes]:https://github.com/hyperhq/hypernetes
 [Hyper.sh]:https://www.hyper.sh
+
+## DaoCloud.io
+
+- *Application*: container management
+- *Launched*: Sep. 2015
+- *Cluster Size*: 1000+ deployments, each deployments contains 3 node cluster.
+- *Order of Data Size*: 100s of Megabytes
+- *Operator*: daocloud.io
+- *Environment*: Baremetal and virtual machines 
+- *Backups*: None, all data can be recreated if necessary.
+
+In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and use etcd to save meta data for service discovery.
```

Comment:
`In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and use etcd to save meta data for service discovery.` 
=>
`In [DaoCloud][DaoCloud], we use Docker and Swarm to deploy and run our applications, and we use etcd to save metadata for service discovery.`?